src/app/[slug]/route.ts

```diff
@@ -14,7 +14,7 @@ export async function GET(req: NextRequest) {
     // Note: we do not `await` this Promise, to ensure we do not delay sending
     //       the heartbeat response while we are still fetching the data.
     // TODO: Replace with a cron job to fetch breach data and icons.
-    getBreaches().then((breaches) => getBreachIcons(breaches));
+    getBreaches();
```

Comment:
Since we have the cron job now, I think we can remove this altogether. (Also line 13, but I couldn't select that in the PR UI :/)

```suggestion
```

Reply:
fixed